scripts/change-contract-uri.ts

```diff
@@ -13,14 +13,16 @@ program
   )
   .argument("<CONTRACT_URI>", "Contract URI to set in the seller's voucher")
   .option("-e, --env <ENV_NAME>", "Target environment", "testing")
+  .option("-cid, --configId <CONFIG_ID>", "Config id", "testing-80001-0")
```

Comment:
"cid" is confusing with IPFS terms, I think.
I suggest using -c for the short option identifier
```suggestion
  .option("-c, --configId <CONFIG_ID>", "Config id", "testing-80001-0")
```